tests/datalab/issue_manager/test_duplicate.py

```diff
@@ -80,3 +104,49 @@ def test_report(self, issue_manager, embeddings):
             verbosity=3,
         )
         assert "Additional Information: " in report
+
+    @given(embeddings=embeddings_strategy())
+    def test_near_duplicates_are_symmetric(self, embeddings):
+
+        data = {"metadata": ["" for _ in range(len(embeddings))]}
+        lab = Datalab(data)
+        issue_manager = NearDuplicateIssueManager(
+            datalab=lab,
+            metric="euclidean",
+            k=2,
+        )
+        embeddings = np.array(embeddings)
+        issue_manager.find_issues(features=embeddings)
+        near_duplicate_sets = issue_manager.info["near_duplicate_sets"]
+        all_symmetric = all(
+            i in near_duplicate_sets[j]
+            for i, near_duplicates in enumerate(near_duplicate_sets)
+            for j in near_duplicates
+        )
+        assert all_symmetric, "Some near duplicate sets are not symmetric"
+
+    @given(embeddings=embeddings_strategy())
+    def test_near_duplicate_sets_for_issues(self, embeddings):
+        data = {"metadata": ["" for _ in range(len(embeddings))]}
+        lab = Datalab(data)
+        issue_manager = NearDuplicateIssueManager(
+            datalab=lab,
+            metric="euclidean",
+            k=2,
+        )
+        embeddings = np.array(embeddings)
+        issue_manager.find_issues(features=embeddings)
+        issues = issue_manager.issues["is_near_duplicate_issue"]
+        near_duplicate_sets = issue_manager.info["near_duplicate_sets"]
+
+        # if an example is not a near duplicate issue, then the near_duplicate_set should be empty and vice versa
+        assert all(
+            len(near_duplicate_set) == 0
+            for i, near_duplicate_set in enumerate(near_duplicate_sets)
+            if not issues[i]
+        ), "Non-issue examples should not have near duplicate sets"
+        assert all(
+            len(near_duplicate_set) > 0
+            for i, near_duplicate_set in enumerate(near_duplicate_sets)
+            if issues[i]
+        ), "Issue examples should have near duplicate sets"
```

Comment:
think you could easily combine these into one test for runtime efficiency, but not important if it's not easy to do

Reply:
Combined tests as they share the same setup code.